cluster/validate-cluster.sh

```diff
@@ -62,9 +62,6 @@ for (( i=0; i<${#MINION_NAMES[@]}; i++)); do
     if [[ "${count}" == "0" ]]; then
       echo -e "${color_red}Failed to find ${MINION_NAMES[$i]}, cluster is probably broken.${color_norm}"
       exit 1
-    else
-      echo -e " ${color_green}[working]${color_norm}"
-      break
```

Comment:
For GCE I observe:
```
satnam@satnam:~/gocode/src/github.com/GoogleCloudPlatform/kubernetes$ cluster/validate-cluster.sh 
Project: kubernetes-satnam
Project: kubernetes-satnam
Running: cluster/../cluster/gce/../../cluster/../cluster/gce/../../_output/dockerized/bin/linux/amd64/kubectl get minions -o template -t {{range.items}}{{.id}}
{{end}}
Found 4 nodes.
Attempt 1 at checking Kubelet installation on node kubernetes-minion-1 ... [working]
Attempt 1 at checking Kubelet installation on node kubernetes-minion-2 ... [working]
Attempt 1 at checking Kubelet installation on node kubernetes-minion-3 ... [working]
Attempt 1 at checking Kubelet installation on node kubernetes-minion-4 ... [working]
Cluster validation succeeded
satnam@satnam:~/gocode/src/github.com/GoogleCloudPlatform/kubernetes$
```
For GKE I saw:
```
Found 2 nodes.
Attempt 1 at checking Kubelet installation on node k8s-satnam-gke-node-1 ... [working]
Attempt 1 at checking Kubelet installation on node k8s-satnam-gke-node-2 ... [working]
Cluster validation succeeded
```
Exactly what is not being printed? Sorry for being so dim.